prophet/prophet.go

```diff
@@ -61,6 +67,34 @@ func (p *P) Run() error {
 		return fmt.Errorf("failed to run votes loop: %w", err)
 	}
 
+	go func() {
+		client, err := client.NewClientFromNode(tendermintRpc)
+		if err != nil {
+			panic(err)
+		}
+
+		timeout := time.After(5 * time.Minute)
```

Comment:
i think we can give up far earlier, if the server isn't reachable almost immediately then it's never gonna be

```suggestion
		timeout := time.After(30 * time.Second)
```